.travis.yml

```diff
@@ -112,16 +115,14 @@ jobs:
       # Deploy built docs : a) when PR is merged to master, b) when a tag is built
       deploy:
         - provider: pages
-          skip-cleanup: true
```

Comment:
@anmolsjoshi shouldn't we still keep `skip_cleanup` as `cleanup` as suggested in the warning:
```
jobs.include.deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
```

Reply:
@vfdev-5 have a look [here](https://blog.travis-ci.com/2019-08-27-deployment-tooling-dpl-v2-preview-release) under Cleaning up the Git working directory.

skip_cleanup has been deprecated to cleanup. But cleanup is False by default which is the same behavior as skip_cleanup=True.

So I think removing it is fine. Thoughts?

Reply:
Okay for me too. Thanks